license_manager/apps/subscriptions/management/commands/send_license_utilization_emails.py

```
@@ -0,0 +1,59 @@
+import logging
+
+from django.core.management.base import BaseCommand
+
+from license_manager.apps.api.tasks import (
+    send_utilization_threshold_reached_email_task,
+    send_weekly_utilization_email_task,
+)
+from license_manager.apps.api_client.enterprise import EnterpriseApiClient
+from license_manager.apps.subscriptions.models import SubscriptionPlan
+from license_manager.apps.subscriptions.utils import localized_utcnow
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = (
+        'Send email alerts to enterprise admins about license utilization of plans with auto-applied licenses.'
+    )
+
+    def handle(self, *args, **options):
+        now = localized_utcnow()
+
+        subscriptions = SubscriptionPlan.objects.filter(
+            should_auto_apply_licenses=True,
+            is_active=True,
+            start_date__lte=now,
+            expiration_date__gte=now
+        ).select_related('customer_agreement')
+
+        if not subscriptions:
+            logger.info('No subscriptions with auto-applied licenses found, skipping license-utilization notifications.')
+            return
+
+        api_client = EnterpriseApiClient()
+
+        for subscription in subscriptions:
+            enterprise_customer_uuid = subscription.customer_agreement.enterprise_customer_uuid
+            admin_users = api_client.get_enterprise_admin_users(enterprise_customer_uuid)
+            email_recipients = [
+                {
+                    'ecu_id': user['ecu_id'],
+                    'email': user['email'],
+                    'created': user['created']
+                }
+                for user in admin_users
+            ]
+
+            # logging is already handled by tasks
+            try:
+                send_weekly_utilization_email_task(subscription, email_recipients)
```

should this be async `send_weekly_utilization_email_task.delay(subscription, email_recipients)`? and if so, i think we can probably drop the except: pass, especially where we're not logging anything

---

yeah I think these can be async for sure